app/src/main/java/com/amaze/filemanager/ui/fragments/AppsListFragment.java

```diff
@@ -136,6 +137,20 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
     }
   }
 
+  @Override
+  public boolean onOptionsItemSelected(@NonNull MenuItem item) {
+    switch (item.getItemId()) {
+      case R.id.sort:
+        showSortDialog(((MainActivity) requireActivity()).getAppTheme());
+        return true;
+      case R.id.exit:
+        ((MainActivity) requireActivity()).goToMain(null);
```

Comment:
Exit should not go to MainFragment, it should finish MainActivity.

Reply:
I know this is previous behaviour, but isn't it weird?